documentation/existing.md

```diff
@@ -2,16 +2,22 @@
 title: Configurations on Existing Kubernetes Cluster
 ---
 
-If a Kubernetes cluster has not been deployed using Kubernetes-Anywhere, follow the instructions below to enable the vSphere Cloud Provider. These steps are not needed when using Kubernetes-Anywhere, they will be done as part of the deployment.
+# Enabling vSphere cloud provider
```

Comment:
Let's use the capital letter consistently for the term "vSphere Cloud Provider". Same for other places.